README.md

```diff
@@ -176,7 +177,7 @@ To generate a language currently not supported by Looker's SDK code generator wi
 
 - configure the desired language in [`codeGenerators.ts`](packages/sdk-codegen/src/codeGenerators.ts). Currently, only Go and C# have legacy language generation configured, and we now have a prototype Look# SDK that can be used instead of the legacy C# generator.
 
-- the legacy generator defaults to using the API 4.0 specification, which is more accurate for strongly typed languages. To use API 3.1, put `api_version=3.1` in the `Looker` section of your `looker.ini`
+- the legacy generator defaults to using the API 4.0 specification. To use deprecated API 3.1, specify `api_version=3.1` in the `Looker` section of your `looker.ini`
```

Comment:
Did you actually test this? I haven't tried the legacy generator in literally a year or more.

Reply:
I tested it and it threw an opaque error from sdk-rtl/lib/transport.js:221... although I couldn't immediately tell if this was a resolvable network/instance/config issue, it seems like your comment is suggesting that this section should be removed, so I've tentatively removed it.

Reply:
I'm good with removing it